flang/test/Lower/HLFIR/vector-subscript-as-value.f90

```diff
@@ -1,6 +1,6 @@
 ! Test lowering of vector subscript designators outside of the
 ! assignment left-and side and input IO context.
-! RUN: bbc -emit-hlfir -o - -I nw %s 2>&1 | FileCheck %s
+! RUN: bbc -emit-hlfir -o - -I nw %s --polymorphic-type 2>&1 | FileCheck %s
```

Comment:
Not for this commit, but I think we should keep the defaults in bbc in line with flang-new. As I understand it, polymorphism is enabled by default now?